src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023imaginginterface.py

```diff
@@ -0,0 +1,123 @@
+from dateutil.parser import parse as dateparse
+import datetime
+import json
+import glob
+
+from abdeladim_2023imagingextractor import (
+    Abdeladim2023MultiPlaneImagingExtractor,
+    Abdeladim2023SinglePlaneImagingExtractor,
+)
+from neuroconv.datainterfaces.ophys.baseimagingextractorinterface import BaseImagingExtractorInterface
+from neuroconv.utils import FolderPathType
+from neuroconv.utils.dict import DeepDict
+from roiextractors.extractors.tiffimagingextractors.scanimagetiff_utils import extract_extra_metadata
+from roiextractors.extractors.tiffimagingextractors.scanimagetiffimagingextractor import ScanImageTiffSinglePlaneImagingExtractor
+
+class Abdeladim2023SinglePlaneImagingInterface(BaseImagingExtractorInterface):
+    """
+    Data Interface for writing imaging data for the MouseV1 to NWB file using Abdeladim2023SinglePlaneImagingExtractor.
+    """
+
+    Extractor = Abdeladim2023SinglePlaneImagingExtractor
+
+    @classmethod
+    def get_available_channels(cls, folder_path):
+        file_path = sorted(glob.glob(f"{folder_path}/*.tif"))
+        return ScanImageTiffSinglePlaneImagingExtractor.get_available_channels(file_path=file_path[0])
+
+    @classmethod
+    def get_available_planes(cls, folder_path):
+        file_path = sorted(glob.glob(f"{folder_path}/*.tif"))
+        return ScanImageTiffSinglePlaneImagingExtractor.get_available_planes(file_path=file_path[0]) 
+
+    def __init__(
+        self,
+        folder_path: FolderPathType,
+        channel_name: str,
+        plane_name: str,
+        verbose: bool = True,
+    ):
+        self.channel_name = channel_name
+        self.plane_name = plane_name
+
+        file_path = sorted(glob.glob(f"{folder_path}/*.tif"))
```

Comment:
I would suggest this everywhere where you do this naming: `file_path` would indicate a single path or string, I would suggest to use `file_paths`

Reply:
I will change this in #5

Reply:
I will change this in #5